lib/config.js

```diff
@@ -548,3 +550,15 @@ function loadConfigFile (configFile) {
 
   return null
 }
+
+function loadServerCaCertFile (opts) {
+  if (opts.serverCaCertFile) {
+    try {
+      return fs.readFileSync(opts.serverCaCertFile)
+    } catch (err) {
+      opts.logger.error('Elastic APM initialization error: Can\'t read server CA cert file %s', opts.serverCaCertFile)
+      opts.logger.error('APM Server transport error:', err.stack)
```

Comment:
The text here is probably just due to a copy-paste error. And since the stack trace isn't important here, I would personally just merge the two lines into something like this:

```js
      opts.logger.error('Elastic APM initialization error: Can\'t read server CA cert file %s (%s)', opts.serverCaCertFile, err.message)
```

Or do as in the `loadConfigFile` function above and just output the stack trace standalone.

Reply:
yikes :)
done, thanks!